is_core/forms/widgets.py

```diff
@@ -135,14 +126,8 @@ def render(self, name, value, attrs={}):
 
 class DragAndDropImageInput(DragAndDropFileInput):
 
-    def _get_thumbnail(self, value):
-        if not default:
-            raise ImproperlyConfigured('Please install sorl.thumbnail before using drag-and-drop file input')
-        return default.backend.get_thumbnail(value, '64x64', crop='center')
-
     def _render_value(self, value):
-        thumbnail = self._get_thumbnail(value)
-        return '<img src="%s" alt="%s">' % (thumbnail.url, thumbnail.name)
+        return ''
```

Comment:
tohle je nějaké divné, co to dělá? Má cenu to přetěžovat?

Reply:
odpovím si, nemá, teoreticky celý ten DragAndDropImageInput může jít do koše

Reply:
a používat předka

Reply:
Když se používá předek, tak tam je potom zdvojená ta cesta souboru.
![image](https://user-images.githubusercontent.com/2447438/62192030-21bb9d80-b375-11e9-855a-e910590fb49e.png)